### PR TITLE
dev, v3.0: add how to run TPC-C test

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? 

Add How to test TiDB using TPC-C

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1514

### Which version does your change affect? 

- dev
- v3.0

### Checklist 

- [ ] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by morgo** [comment]
The current file is not in the TOC. Did you want to move it under `/reference/performance`? This seems like a possible place (although it is a how-to; we could also move it under `How-to > Scale` )

**Comment by lilin90** [comment]
@morgo Thanks for your reminder. The `benchmark` files are not added to TOC on purpose. Currently, we'll keep them only in the repo.

File: dev/benchmark/how-to-run-tpcc.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+git clone -b 5.0-mysql-support-opt https://github.com/pingcap/benchmarksql.git
+```
+
+To install Java and Ant, run the following command (take CentOS as an example):
+
+```shell
+sudo yum install -y java ant
+```
+
+Come to the `benchmarksql` directory and run `ant` to build:
```
Comment (dcalvin) [inline dev/benchmark/how-to-run-tpcc.md]:
```suggestion
Enter the `benchmarksql` directory and run `ant` to build:
```

File: dev/benchmark/how-to-run-tpcc.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+## Run the test
+
+Run the following BenchmarkSQL test script:
+
+```shell
+nohup ./runBenchmark.sh props.mysql &> test.log &
+```
+
+After the execution is finished, view the result using the `test.log`:
```
Comment (dcalvin) [inline dev/benchmark/how-to-run-tpcc.md]:
```suggestion
After the execution is finished, view the result using `test.log`:
```

File: v3.0/benchmark/how-to-run-tpcc.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+git clone -b 5.0-mysql-support-opt https://github.com/pingcap/benchmarksql.git
+```
+
+To install Java and Ant, run the following command (take CentOS as an example):
+
+```shell
+sudo yum install -y java ant
+```
+
+Come to the `benchmarksql` directory and run `ant` to build:
```
Comment (dcalvin) [inline v3.0/benchmark/how-to-run-tpcc.md]:
```suggestion
Enter the `benchmarksql` directory and run `ant` to build:
```

File: v3.0/benchmark/how-to-run-tpcc.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+## Run the test
+
+Run the following BenchmarkSQL test script:
+
+```shell
+nohup ./runBenchmark.sh props.mysql &> test.log &
+```
+
+After the execution is finished, view the result using the `test.log`:
```
Comment (dcalvin) [inline v3.0/benchmark/how-to-run-tpcc.md]:
```suggestion
After the execution is finished, view the result using `test.log`:
```

File: v3.0/benchmark/how-to-run-tpcc.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+```text
+07:09:53,455 [Thread-351] INFO   jTPCC : Term-00, Measured tpmC (NewOrders) = 77373.25
+07:09:53,455 [Thread-351] INFO   jTPCC : Term-00, Measured tpmTOTAL = 171959.88
+07:09:53,455 [Thread-351] INFO   jTPCC : Term-00, Session Start     = 2019-03-21 07:07:52
+07:09:53,456 [Thread-351] INFO   jTPCC : Term-00, Session End       = 2019-03-21 07:09:53
+07:09:53,456 [Thread-351] INFO   jTPCC : Term-00, Transaction Count = 345240
+```
+
+The tpmC section is the testing result.
```
Comment (dcalvin) [inline v3.0/benchmark/how-to-run-tpcc.md]:
```suggestion
The value in the tpmC section is the testing result.
```

File: dev/benchmark/how-to-run-tpcc.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+```text
+07:09:53,455 [Thread-351] INFO   jTPCC : Term-00, Measured tpmC (NewOrders) = 77373.25
+07:09:53,455 [Thread-351] INFO   jTPCC : Term-00, Measured tpmTOTAL = 171959.88
+07:09:53,455 [Thread-351] INFO   jTPCC : Term-00, Session Start     = 2019-03-21 07:07:52
+07:09:53,456 [Thread-351] INFO   jTPCC : Term-00, Session End       = 2019-03-21 07:09:53
+07:09:53,456 [Thread-351] INFO   jTPCC : Term-00, Transaction Count = 345240
+```
+
+The tpmC section is the testing result.
```
Comment (dcalvin) [inline dev/benchmark/how-to-run-tpcc.md]:
```suggestion
The value in the tpmC section is the testing result.
```

**Pull request merged**